### PR TITLE
[12.0][MIG] sale_product_multi_add

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @Eficent

### CODE REVIEW

File: sale_product_multi_add/wizards/sale_import_products_view.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    
+    <record id="view_import_product_to_sale" model="ir.ui.view">
+        <field name="name">import.products.to.sale.form</field>
+        <field name="model">sale.import.products</field>
+        <field name="arch" type="xml">
+        <form string="Products">
+        <field name="products" />
+        <footer>
```
Comment (TimLai125) [inline sale_product_multi_add/wizards/sale_import_products_view.xml]: It will be nice if the indentation can be fixed in this xml file.

**Review by TimLai125 — CHANGES_REQUESTED** [review]

File: sale_product_multi_add/wizards/sale_import_products_view.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply sale_product_multi_add/wizards/sale_import_products_view.xml]: Thanks for code review :smiley:

**Comment by lfreeke** [comment]
Is there a reason why the line number will be 9,999 when I add the products via the add multiple products wizard?

**Review by HaraldPanten — APPROVED** [review]
Functional review OK, with the exception of the sale order line sequence.

File: sale_product_multi_add/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Cédric Pigeon, ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Sale Product Multi Add",
+    'summary': """
+        Sale Product Multi Add """,
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```
Comment (nikul-serpentcs) [inline sale_product_multi_add/__manifest__.py]: @hveficent  URL should be `https://github.com/OCA/sale-workflow`

**Comment by HviorForgeFlow** [comment]
> Is there a reason why the line number will be 9,999 when I add the products via the add multiple products wizard?

It is related with definition of sequence in sale.order.line on sale_order_line_sequence module.
The default is set to 9999 and as this module create sale order lines from a wizard it uses the default sequence in all products. What I noticed also is that if you confirm the order that sequence is rebuild properly.

File: sale_product_multi_add/__manifest__.py (hunk 2 above)
Comment (HviorForgeFlow) [reply sale_product_multi_add/__manifest__.py]: Done

**Comment by lfreeke** [comment]
> > Is there a reason why the line number will be 9,999 when I add the products via the add multiple products wizard?
> 
> It is related with definition of sequence in sale.order.line on sale_order_line_sequence module.
> The default is set to 9999 and as this module create sale order lines from a wizard it uses the default sequence in all products. What I noticed also is that if you confirm the order that sequence is rebuild properly.

It looks a little bit weird but if it rebuilds after confirm i'm fine with it :+1:

**Review by lfreeke — APPROVED** [review]
Approve (functional test)

**Comment by HviorForgeFlow** [comment]
@TimLai125 could you update your review? thanks.

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**